packages/mrm/src/requirex.js

```diff
@@ -0,0 +1,33 @@
+const npx = require('libnpx');
+const path = require('path');
+
+async function requirex_(packageName) {
```

Comment:
I hope my answer to the previous comment answers this as well, otherwise I can try to elaborate.

Reply:
And here the question about composing file paths — it looks brittle.

Reply:
I was confused as the comment was pointing to the line above. I pulled this out of the [internal implementation](https://github.com/npm/npx/blob/latest/bin/index.js#L6) of `npx` with the only change being the addition of `..` as the script is under `src/`.

I'm not a fan either and open ot any suggestions.

Reply:
I think we can use something like https://github.com/timoxley/npm-which since npm is a dependency.

Reply:
Maybe we can remove the `npm` dependency altogether, since `process.argv[0]` will always be `node` and it's always bundled with `npm`. I'll try this and if it doesn't work I'll give `npm-which` a shot.

Reply:
Ended up using `process.execPath` which is meant for this and is more robust. Worked like a charm!